app/components/collection-search.cjsx

```diff
@@ -23,7 +23,7 @@ module.exports = React.createClass
     query =
       page_size: 20
       favorite: false
-      current_user_roles: 'owner,collaborator,contributor,viewer'
+      current_user_roles: 'owner,collaborator,contributor'
```

Comment:
For some reason this isn't showing up in the diff view on github which is a bit concerning.